.github/workflows/ci.yml

```diff
@@ -21,6 +21,8 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ['3.10']
+        split_size: [4]
+        group_number: [1, 2, 3, 4]
```

Comment:
this is where we can decide how many nodes we want to use, right?

Reply:
Correct